administrator/components/com_guidedtours/src/Model/ToursModel.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+/**
+ * @package       Joomla.Administrator
+ * @subpackage    com_guidedtours
+ * @copyright (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
```

Comment:
The indentation here (and elsewhere is wrong. 
The (C) should be in line with com_ above